frontend/assets/style/components/_forms.scss

```diff
@@ -418,22 +418,22 @@ p.error {
 
     &::before {
       content: "";
-      width: 14px;
-      height: 14px;
+      width: 0.875rem;
+      height: 0.875rem;
       margin-right: 0.5rem;
       border: 1px solid $general_0;
-      transform: translateY(2px);
+      transform: translateY(0.125rem);
       transition: box-shadow 150ms, background-color 150ms;
     }
 
     &::after {
       @include icon(check);
-      font-size: 10px;
+      font-size: 0.625rem;
       position: absolute;
-      width: 6px;
-      height: 6px;
-      top: 9px;
-      left: 5px;
+      width: 0.375rem;
+      height: 0.375rem;
+      top: 0.5625rem;
+      left: 0.3125rem;
```

Comment:
In this component, let's keep the px. Using `rem` doesn't work well because things start being missaligned :/

Reply:
Unfortunately if we leave some element in px, they will not scale with the rest

Reply:
I see. It's a tradeoff. But I guess that not scaling a radio button isn't a big deal.